src/components/PostList/PostList.tsx

```diff
@@ -1,3 +1,16 @@
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+import { Post } from '../../types/Post';
+import { PostInfo } from '../PostInfo';
+
+type Props = {
+  posts: Post[],
+};
+
+export const PostList: React.FC<Props> = ({ posts }) => (
+  <div className="PostList">
+    {posts.map(post => (
+      <div className="PostInfo" key={post.id}>
+        <PostInfo post={post} />
```

Comment:
```suggestion
        <PostInfo post={post} key={post.id} />
```

you need `key` when you generate lists